xbmc/addons/PVRClient.cpp

```diff
@@ -785,6 +786,46 @@ PVR_ERROR CPVRClient::IsRecordable(const EPG::CConstEpgInfoTagPtr &tag, bool *is
   return retVal;
 }
 
+bool CPVRClient::IsPlayable(const EPG::CConstEpgInfoTagPtr &tag)
+{
+  if (!m_bReadyToUse)
+    return false;
+
+  try
+  {
+    EPG_TAG pvrTag;
+    WriteEpgTag(tag, pvrTag);
+    return m_pStruct->IsPlayable(pvrTag);
+  }
+  catch (std::exception &e)
+  {
+    LogException(e, __FUNCTION__);
+  }
+
+  return false;
+}
+
+const std::string CPVRClient::GetEpgTagUrl(const EPG::CConstEpgInfoTagPtr &tag)
+{
+  if (!m_bReadyToUse)
+    return NULL;
```

Comment:
We're doing c++, thus "return nullptr;" No need to change all the old code still using NULL, but for new code we should use "modern" c++ features.